src/main/java/seedu/address/model/manager/ServiceManager.java

```diff
@@ -8,25 +8,26 @@
 
 import javafx.collections.ObservableList;
 import seedu.address.model.service.Service;
-import seedu.address.model.util.uniquelist.UniqueList;
+import seedu.address.model.service.ServiceCode;
+import seedu.address.model.service.UniqueServiceList;
 
 /**
  * Wraps all data at the ServiceManager level
  * Duplicates are not allowed (by .equals comparison)
  */
 public class ServiceManager implements ReadOnlyServiceManager {
 
-    private final UniqueList<Service> services;
+    private final UniqueServiceList services;
 
     public ServiceManager() {
-        this.services = new UniqueList<>();
+        this.services = new UniqueServiceList();
     }
```

Comment:
I think Hans made a generic list?

Reply:
Yup he did, but I needed to extend it with more methods. I need a contains method with parameter ServiceCode to check if the service code belongs to an existing service. I can't just insert that as a generic method to Unique List, so I created UniqueServiceList to extend UniqueList<Service>.